Not sure if I should to roll-out all the change in the same commit/bug since each one fixes a different build error introduced in different commits.

In particular, this commit fixes this build error in WPEWebKit/WebKitGTK:  
 
```
                 from webkit/DerivedSources/WebKit/unified-sources/UnifiedSource-13fb4697-1.cpp:1:
webkit/WebCore/PrivateHeaders/WebCore/DMABufReleaseFlag.h: In member function 'void WebCore::DMABufReleaseFlag::release()':
webkit/WebCore/PrivateHeaders/WebCore/DMABufReleaseFlag.h:77:27: error: invalid initialization of non-const reference of type 'WTF::TextStream&' from an rvalue of type 'int'
   77 |         if (write(fd.value(), &value, sizeof(value)) != sizeof(value))
      |                   ~~~~~~~~^~
In file included from webkit/DerivedSources/WebKit/WebProcessMessages.h:43,
                 from webkit/Source/WebKit/GPUProcess/GPUConnectionToWebProcess.cpp:69,
                 from webkit/DerivedSources/WebKit/unified-sources/UnifiedSource-13fb4697-1.cpp:1:
webkit/WebCore/PrivateHeaders/WebCore/RenderTreeAsText.h:61:12: note: in passing argument 1 of 'void WebCore::write(WTF::TextStream&, const WebCore::RenderObject&, WTF::OptionSet<WebCore::RenderAsTextFlag>)'
   61 | void write(WTF::TextStream&, const RenderObject&, OptionSet<RenderAsTextFlag> = { });
      |            ^~~~~~~~~~~~~~~~
```

In `/WebCore/PrivateHeaders/WebCore/RenderTreeAsText.h` there  is redefinition of `void write(WTF::TextStream&, const RenderObject&, OptionSet<RenderAsTextFlag> = { });`:

``` cpp
...
void write(WTF::TextStream&, const RenderObject&, OptionSet<RenderAsTextFlag> = { });
void writeDebugInfo(WTF::TextStream&, const RenderObject&, OptionSet<RenderAsTextFlag> = { });
...
```

 when the function `ssize_t write(int, const void*, size_t)` from `unistd.h` is expected in `WebCore/PrivateHeaders/WebCore/DMABufReleaseFlag.h:77`.

--- a/Source/WebCore/platform/graphics/gbm/DMABufReleaseFlag.h
+++ b/Source/WebCore/platform/graphics/gbm/DMABufReleaseFlag.h
@@ -74,7 +74,7 @@ struct DMABufReleaseFlag {
             return;
 
         uint64_t value { 1 };
-        if (write(fd.value(), &value, sizeof(value)) != sizeof(value))
+        if (::write(fd.value(), &value, sizeof(value)) != sizeof(value))
             WTFLogAlways("Error writing to the eventfd at DMABufReleaseFlag: %s", safeStrerror(errno).data());
     }
 

